zio/detector/file.go

```diff
@@ -49,7 +48,7 @@ func OpenFile(zctx *resolver.Context, path string, cfg OpenConfig) (*zbuf.File,
 	}
 
 	var f *os.File
-	if cfg.DashStdin && path == "-" {
+	if path == "/dev/stdin" {
```

Comment:
Zap just does "stdin" for this case- I don't think `/dev/stdin` makes sense for windows right?

Reply:
This is just an internal string. (I'll define it as a constant somewhere). The only requirement is that it shouldn't conflict with an actual file name. Are you concerned that we could have a file called "/dev/stdin" on Windows?

Reply:
No, this is fine.

Reply:
I might consider adding this as an export const `StdinPath` or something and using this in `zq/zq/cmd.go`

Reply:
indeed - done